app/services/pwned_password.rb

```diff
@@ -0,0 +1,35 @@
+class PwnedPassword
+  RECORD_SIZE = 41
+
+  def initialize(file_name)
+    @file_name = file_name
+  end
+
+  # :reek:TooManyStatements
+  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/AbcSize
+  def call(password)
```

Comment:
Since this could be a little tough to follow, wanna drop a comment in here explaining what is going on?